src/main/webapp/app/instructor-dashboard/instructor-dashboard-archive-dialog.component.html

```diff
@@ -1,13 +1,13 @@
 <form name="archiveForm" role="form" (ngSubmit)="confirmArchive(exercise.id)" #archiveForm="ngForm">
     <div class="modal-header">
-        <h4 class="modal-title" jhiTranslate="entity.archive.title">Confirm archive operation</h4>
+        <h4 class="modal-title" jhiTranslate="entity.exportRepos.warning">Confirm Export</h4>
```

Comment:
I think this is a mistake and you accidentally changed this file. Please revert it.